### PR TITLE
add 3 libraries clapack, distarray and ossuuid

### PR DESCRIPTION
**Pull request by dnadeau4**
UV-CDAT needs these libraries

### CODE REVIEW

File: recipes/clapack/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
```
Comment (ocefpaf) [inline recipes/clapack/build.sh]: Please try `CPPFLAGS="-I${PREFIX}/include" $CPPFLAGS` in all of these variables to preserve the original settings.

File: recipes/clapack/build.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export LDFLAGS="-L${PREFIX}/lib"
+
+cmake . -DCMAKE_INSTALL_PREFIX=${PREFIX}
+make
```
Comment (ocefpaf) [inline recipes/clapack/build.sh]: Is there a make check?

File: recipes/clapack/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  commands:
+    - test -f $PREFIX/lib/liblapack.a
+    - test -f $PREFIX/lib/libblas.a
+    - test -f $PREFIX/lib/libf2c.a
+
+about:
+  home: http://www.netlib.org/clapack/faq.html
+  license: BSD
```
Comment (ocefpaf) [inline recipes/clapack/meta.yaml]: Can you specify if that is `BSD 3-Clause` or `BSD 2-Clause`?

File: recipes/distarray/build.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC $CFLAGS"
```
Comment (ocefpaf) [inline recipes/distarray/build.sh]: :+1: please do the same in the other env vars.

File: recipes/distarray/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{% set name = "distarray" %}
```
Comment (ocefpaf) [inline recipes/distarray/meta.yaml]: Please remove the tarball.

File: recipes/distarray/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    - setuptools
+  run:
+    - python
+test:
+    imports:
+     - distarray
+
+about:
+  home: http://uvcdat.llnl.gov/
+  license: GPL
```
Comment (ocefpaf) [inline recipes/distarray/meta.yaml]: Do you know the version?

File: recipes/ossuuid/build.sh [hunk 7]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure  --prefix=$PREFIX
+make -j
+make install
```
Comment (ocefpaf) [inline recipes/ossuuid/build.sh]: `make check`?

File: recipes/ossuuid/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libuuid.a
+    - test -f $PREFIX/lib/libuuid.so
+
+about:
+  home: ftp://ftp.ossp.org/pkg/lib/uuid/
+  license: GPL
```
Comment (ocefpaf) [inline recipes/ossuuid/meta.yaml]: versoin?

File: recipes/clapack/build.sh (hunk 2 above)
Comment (dnadeau4) [inline recipes/clapack/build.sh]: I found a check in the `CMAKEFILE`, I will add it.
`make cmake_check_build_system`

**Comment by dnadeau4** [comment]
@ocefpaf All test have passed.  If you are ready to push this to `feedstock`, I will work on `libcf` whcih needs some dependencies from this PR.

**Pull request merged**